code/modules/mechs/_mech_setup.dm

```diff
@@ -39,6 +39,8 @@
 #define FRAME_WIRED							1
 #define FRAME_WIRED_ADJUSTED				2
 
+#define STANDARD_MECH_MOUNTED_WEAPON_VARS use_external_power = TRUE;restrict_safety = TRUE;twohanded = FALSE;self_recharge = TRUE
```

Comment:
This is very messy and severely hurts readability. There is no reason to do this over just setting the vars the normal way as they already  were.

Reply:
I think isn't cool to leave many code repetitions
But if u talk about tabulation and <br> in define... 🤔